src/twisted/conch/interfaces.py

```diff
@@ -363,8 +364,8 @@ def matchesHost(hostname):
         address, you have to resolve it yourself, and pass it in as a dotted
         quad string.
 
-        @param key: The hostname to match against.
-        @type key: L{str}
+        @param hostname: The hostname to match against.
+        @type hostname: L{str}
```

Comment:
It looks like this is `L{str} or L{bytes}` in the `PlainHosts` and `HashedHosts` implementations (though their docs claim just `L{bytes}`). I filed https://twistedmatrix.com/trac/ticket/9713 for this.